ibm_catalog.json

```diff
@@ -0,0 +1,94 @@
+{
+  "products": [
+    {
+      "name": "deploy-arch-ibm-secrets-manager",
+      "label": "Secrets Manager DA",
+      "product_kind": "solution",
+      "tags": [
+        "ibm_created",
+        "target_terraform",
+        "terraform",
+        "solution"
+      ],
+      "keywords": [
+        "Secrets",
+        "Secrets Manager",
+        "IaC",
+        "infrastructure as code",
+        "terraform",
+        "solution"
+      ],
+      "short_description": "Creates a Secrets Manager instance.",
+      "long_description": "This module is used to provision and configure an IBM Cloud Secrets Manager instance.",
+      "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-secrets-manager/blob/main/README.md",
```

Comment:
should point to the README in the DA directory

Reply:
There's no README in the DA directory... Generally, we give module's readme in the ibm_catalog.json